.config/credentials.json.template

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
@AdamSulek your commit is eliminating this file from the `main` branch. Please uncommit this file from your PR (or just add it back and commit again). In general be careful using `git commit add .` as it adds all your repo changes, it is better to add folders or files `git commit add tests`